databuilder/databuilder/extractor/glue_extractor.py

```diff
@@ -25,6 +25,7 @@ def init(self, conf: ConfigTree) -> None:
         conf = conf.with_fallback(GlueExtractor.DEFAULT_CONFIG)
         self._cluster = conf.get_string(GlueExtractor.CLUSTER_KEY)
         self._filters = conf.get(GlueExtractor.FILTER_KEY)
+        self._max_results = 500
```

Comment:
should we make it as a configurable which default as 500?

Reply:
@feng-tao I can do it the same way as we define `Filters`. However, we discussed the implementation with Grant Seward (Stemma) so he assumed it might be not required so a hardcode would work fine.
If you want, I can make it available for passing the param via config.
What do you think?

Reply:
yeah, let's make it as a config.